protocol/read-write-resource/post-uri-assignment-slug.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Assignment of URIs for POST requests with Slug
+
+  Background: Set up container
+    * def testContainer = rootTestContainer.createContainer()
+    
+  Scenario: Allow resource creation via POST with Slug
+    * def containerUrl = testContainer.url
+    Given url containerUrl
+    And headers clients.alice.getAuthHeaders('POST', containerUrl)
```

Comment:
```suggestion
    Given url testContainer.url
    And headers clients.alice.getAuthHeaders('POST', testContainer.url)
```
Now that we are using the bean properties I wonder if it is cleaner to drop the variable definition and just use the property in each location. What do you think?